curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,234 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
 
 #### --text--
 
-Which property changes the background color in CSS?
+Why do you need to have good color contrast on your webpage?
 
 #### --distractors--
 
-`bg-color`
+To make the page more vibrant.
 
 ---
 
-`color-bg`
+To meet requirements for search engine optimization (SEO).
 
 ---
 
-`fill-color`
+To make important elements of the page stand out.
 
 #### --answer--
 
-`background-color`
+To make the page content accessible and readable.
 
 ### --question--
 
 #### --text--
 
-Which CSS property controls the width of an element?
+Which of the following tools allows you to enter background and foreground colors and check their contrast ratio?
 
 #### --distractors--
 
-`size`
+TPGi Color Contrast Analyzer.
 
 ---
 
-`dimension`
+Figma.
```

Comment:
We have decided to not use a period (`.) for single worded options.
For consistency, it is good to remove period from all the single worded options in this quiz.
Other than that everything seems good to go.